bin/reth/src/commands/stage/run.rs

```diff
@@ -284,6 +286,7 @@ impl Command {
                 break
             }
         }
+        info!(target: "reth::cli", stage= ?self.stage, time = ?start.elapsed().as_secs(), "Finished stage.");
```

Comment:
the debug impls already formats properly
```suggestion
        info!(target: "reth::cli", stage= ?self.stage, time = ?start.elapsed(), "Finished stage.");
```